src/main/java/com/oheers/fish/EvenMoreFish.java

```diff
@@ -268,29 +272,29 @@ private void optionalListeners() {
         }
     }
 
-    private void metrics() {
-        Metrics metrics = new Metrics(this, METRIC_ID);
-
-        metrics.addCustomChart(new SingleLineChart("fish_caught", () -> {
-            int returning = metric_fishCaught;
-            metric_fishCaught = 0;
-            return returning;
-        }));
-
-        metrics.addCustomChart(new SingleLineChart("baits_applied", () -> {
-            int returning = metric_baitsApplied;
-            metric_baitsApplied = 0;
-            return returning;
-        }));
-
-        metrics.addCustomChart(new SingleLineChart("baits_used", () -> {
-            int returning = metric_baitsUsed;
-            metric_baitsUsed = 0;
-            return returning;
-        }));
-
-        metrics.addCustomChart(new SimplePie("experimental_features", () -> mainConfig.doingExperimentalFeatures() ? "true" : "false"));
-    }
+    //private void metrics() {
```

Comment:
Why is the metrics code commented out? Does it not work?

Reply:
Yeah it's currently not working. I need to figure out why when I have the time.